main.tf

```diff
@@ -5,7 +5,7 @@ resource "oci_logging_log_group" "apigwloggroup" {
 
   compartment_id = var.compartment_id
   description    = "API Gateway Loggroup"
-  display_name   = each.value
+  display_name   = var.label_prefix == "none" ? each.value : format("%s-%s", var.label_prefix, each.value)
```

Comment:
`format("%s-%s", var.label_prefix, each.value)` is repeated a decent number of times in the configuration. I would suggest creating a local and use it in the resource blocks instead: that way, if we choose to change formatting later, it will be a single place update.

That may not work with other format strings that are slightly different (e.g. `format("%s-%s%s", var.label_prefix, "access", each.key)`)

Reply:
good idea.thanks